groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/separations/BastnasiteProcessing.groovy

```diff
@@ -130,49 +129,54 @@ BR.recipeBuilder()
     .EUt(30)
     .buildAndRegister()
 
-CENTRIFUGE.recipeBuilder()
+MIXER_SETTLER.recipeBuilder()
     .fluidInputs(fluid('bast_ox_re_nitrates_solution') * 1000)
     .fluidInputs(fluid('tributyl_phosphate_extraction_mixture') * 10000)
     .fluidOutputs(fluid('bast_ce_free_solution') * 1000)
     .fluidOutputs(fluid('cerium_extract') * 10000)
     .duration(80)
     .EUt(120)
+    .requiredCells(4)
     .buildAndRegister()
 
-CENTRIFUGE.recipeBuilder()
+MIXER_SETTLER.recipeBuilder()
     .fluidInputs(fluid('bast_ce_free_solution') * 1000)
     .fluidInputs(fluid('p_two_zero_four_extraction_mixture') * 300)
     .fluidOutputs(fluid('bast_lree_concentrate') * 1000) // contains 0.036 mol H+
     .fluidOutputs(fluid('bast_lree_free_extract') * 300)
     .duration(80)
     .EUt(120)
+    .requiredCells(8)
     .buildAndRegister()
 
-CENTRIFUGE.recipeBuilder()
+MIXER_SETTLER.recipeBuilder()
     .fluidInputs(fluid('hydrochloric_acid') * 4350)
     .fluidInputs(fluid('bast_lree_free_extract') * 30000) // MREE/HREEs enriched hundredfold
     .fluidOutputs(fluid('bast_lree_free_concentrate') * 4350)
     .fluidOutputs(fluid('spent_p_two_zero_four_extraction_mixture') * 30000)
     .duration(80)
     .EUt(120)
+    .requiredCells(2)
     .buildAndRegister()
 
-CENTRIFUGE.recipeBuilder()
+MIXER_SETTLER.recipeBuilder()
     .fluidInputs(fluid('bast_lree_free_concentrate') * 4350)
     .fluidInputs(fluid('p_five_zero_seven_extraction_mixture') * 5000)
     .fluidOutputs(fluid('bast_mree_concentrate') * 4350) // contains 0.6 mol H+
     .fluidOutputs(fluid('bast_hree_extract') * 5000)
     .duration(80)
     .EUt(120)
+    .requiredCells(8)
     .buildAndRegister()
 
-CENTRIFUGE.recipeBuilder()
+MIXER_SETTLER.recipeBuilder()
     .fluidInputs(fluid('hydrochloric_acid') * 1325)
     .fluidInputs(fluid('bast_hree_extract') * 5000)
     .fluidOutputs(fluid('bast_hree_concentrate') * 1325)
     .fluidOutputs(fluid('spent_p_five_zero_seven_extraction_mixture') * 5000)
     .duration(80)
     .EUt(120)
+    .requiredCells(2)
```

Comment:
Note: I will have to update SuSyCore to allow the mixer settler to use 2 cells; it currently requires a minimum of 4.